giraffe/src/components/BandHoverLayer.tsx

```diff
@@ -103,10 +103,9 @@ export const BandHoverLayer: FunctionComponent<Props> = ({
   })
 
   const colors = []
-  if (Array.isArray(spec.bandIndexMap.rowIndices)) {
-    for (let i = 0; i < spec.bandIndexMap.rowIndices.length; i += 1) {
-      colors.push(spec.scales.fill(i))
-    }
+  for (let i = 0; i < rowIndices.length; i += 1) {
+    const lineIndex = groupColData[rowIndices[i]]
+    colors.push(spec.lineData[lineIndex].fill)
```

Comment:
are lineData and lineIndex guarenteed to exist in here?

Reply:
Oops, yeah good call. I think spec.lineData always exists but lineIndex might not. But we'll just guard against both.